projects/wget2/Dockerfile

```diff
@@ -17,27 +17,40 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER rockdaboot@gmail.com
 RUN apt-get update && apt-get install -y \
+ make \
  pkg-config \
  gettext \
+ autogen \
  autopoint \
  autoconf \
  automake \
  libtool \
  texinfo \
  flex \
- liblzma-dev \
- libidn11-dev \
- libidn2-0-dev \
- libunistring-dev \
- zlib1g-dev \
- libbz2-dev \
- gnutls-dev \
- libpsl-dev \
- libnghttp2-dev \
- doxygen
+ bison \
+ doxygen \
+ gtk-doc-tools \
+ gettext \
+ gengetopt \
+ curl \
```

Comment:
Those are build prerequisites, but I could get rid of doxygen and gtk-doc-tools now. 'python' now has to be installed explicitly.
I also made a few cleanups and fixed a few build issues, static linking seems not very well supported throughout C projects, AC_SEARCH_LIBS 5th param (in configure.ac) is often empty. It is needed only for testing static libraries linkage, naming those libraries that resolves all externals. One can circumvent this by setting $LIBS for the configure run.